source/cfn/templates/buckets.template

```diff
@@ -129,6 +135,9 @@ Resources:
       LoggingConfiguration:
         DestinationBucketName: !Ref AccessLogsBucket
         LogFilePrefix: amplify-storage-bucket/
+      OwnershipControls:
```

Comment:
we are not configuring ACL here, so can probably remove this option and go with default [bucket owner enforced](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3-readme.html#bucket-owner-enforced-recommended)